Rover/mode_auto.cpp

```diff
@@ -40,6 +40,7 @@ void ModeAuto::_exit()
     if (mission.state() == AP_Mission::MISSION_RUNNING) {
         mission.stop();
     }
+    g2.sailboat.clear_in_irons();
```

Comment:
We probably need this on all mode exits not just auto. waypoint navigation is also used in RTL for example.

Reply:
OK. There is an exit class in mode base class. I will put sailboat.clear_in_irons there